app/js/controllers/add-ctrl.js

```diff
@@ -15,12 +15,12 @@ angular
 	$scope.fep = formElementProvider;
 
 	$scope.data = {};
-	$scope.model = {};
+	$scope.model = { attributes: {} };
 
 	$scope.updateResource = function(model, rdesc) {
 		var finalModel = relink(angular.copy(Restangular.stripRestangular(model)), rdesc);
-		delete finalModel.modified;
-		delete finalModel.created;
+		delete finalModel.attributes.modified;
+		delete finalModel.attributes.created;
```

Comment:
I'm not sure why there's an `updateResource` method here at all, given that this is the add controller (shouldn't it be `createResource` or something?) but I think these two `delete` statements aren't needed for the same reason I said they aren't needed for the edit controller back in PR 106 (https://github.com/TechAtNYU/intranet/pull/106/files#r29822699).